SPECS/bpftrace/bpftrace.spec

```diff
@@ -17,7 +17,11 @@ BuildRequires:  elfutils-libelf-devel
 BuildRequires:  flex
 BuildRequires:  gcc
 BuildRequires:  git
-BuildRequires:  llvm-devel >= 8.0.1-5
+BuildRequires:  gmock
```

Comment:
Nit: I'm seeing these new testing dependencies, but no `%check` section. Would it be possible to add tests to this spec?